src/tiledb/cloud/bioimg/ingestion.py

```diff
@@ -350,6 +356,11 @@ def register_dataset_udf(
 
     # serialize udf arguments
     compressor = kwargs.pop("compressor", None)
+
+    # Get either the new experimental or default reader
+    reader = kwargs.pop("reader", "production")
+    experimental_reader = True if reader == "experimental" else False
```

Comment:
Should this be an enum?

Reply:
You are right @ihnorton I changed it in my last commit. I just thought that since the old reader is about to be deprecated once the experimental becomes the default one, that I did not want to over-engineer it. But it is cleaner this way thanks!